### ISSUE TITLE
Vertical scroll bar doesn't appear in Preferences > Plugins

### ISSUE THREAD

**Issue by greyspectrum** [nylas/N1#2928]
##### What operating system are you using?
Ubuntu 14.04 LTS

##### What version of N1 are you using?
0.4.56

##### Do you have any third-party plugins installed? If so, which ones?
No.

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
No.

Under Preferences > Plugins, the vertical scroll bar on the right side sometimes fails to appear when I hover over it, preventing me from scrolling through the plugins.

**Comment by emorikawa** [comment]
@greyspectrum can you still scroll with your mouse-wheel / trackpad on the Plugins pane?

**Comment by greyspectrum** [comment]
Yes. What I'm seeing is: when I first open N1, the scroll bar doesn't appear when I hover over it, but if I use a trackpad scroll, it appears and continues to appear if I hover over it. If I quit N1 and restart, then I'm at the start of the loop again. So, the hover-and-scroll-appears functionality only works after at least one trackpad scroll.
